packages/terra-menu/src/MenuItem.jsx

```diff
@@ -104,6 +108,20 @@ const propTypes = {
    * Indicates if the item should display with a highlighted background. Reserved for Terra higher-order component approved usage only.
    */
   isHighlighted: PropTypes.bool,
+  /**
+   * @private
+   * The index of the menu item.
+   */
+  index: PropTypes.number,
+  /**
+   * @private
+   * Number of items in the menu.
+   */
+  totalItems: PropTypes.number,
+  /**
+   * The intl object to be injected for translations. Provided by the injectIntl function.
+   */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
```

Comment:
should this prop also be private ?

Reply:
Yes, changed to private in https://github.com/cerner/terra-framework/pull/1626/commits/036710d565ae9574da5f7badc1e9678f51cc4f0b